docs/usage.md

```diff
@@ -65,175 +73,207 @@ results         # Finished results (configurable, see below)
-### `--singleEnd`
-By default, the pipeline expects paired-end data. If you have single-end data, you need to specify `--singleEnd` on the command line when you launch the pipeline. A normal glob pattern, enclosed in quotation marks, can then be used for `--reads`. For example:
+### `--peptides`
 
-```bash
---singleEnd --reads '*.fastq'
-```
+Instead of genomic variants, peptide sequences can be provided in a TSV file. In this case, MHC binding predictions will be made for the provided sequences. The TSV file has to include the following columns: `id, sequence`. All additional columns will be added to the prediction output as annotation.
 
-It is not possible to run a mixture of single-end and paired-end files in one run.
+## Additional pipeline parameters
 
+## `--filter_self`
 
-## Reference Genomes
+Specifies that peptides should be filtered against the specified human proteome references. By default, this is turned off.
 
-The pipeline config files come bundled with paths to the illumina iGenomes reference index files. If running with docker or AWS, the configuration is set up to use the [AWS-iGenomes](https://ewels.github.io/AWS-iGenomes/) resource.
+## `--mhc_class`
 
-### `--genome` (using iGenomes)
-There are 31 different species supported in the iGenomes references. To run the pipeline, you must specify which to use with the `--genome` flag.
+Specifies whether the predictions should be done for MHC class I or class II. By default, this is set to 1 (class I).
 
-You can find the keys to specify the genomes in the [iGenomes config file](../conf/igenomes.config). Common genomes that are supported are:
+## `--min_peptide_length`
 
-* Human
-  * `--genome GRCh37`
-* Mouse
-  * `--genome GRCm38`
-* _Drosophila_
-  * `--genome BDGP6`
-* _S. cerevisiae_
-  * `--genome 'R64-1-1'`
+Specifies the minimum peptide length. By default, for MHC Class I this is 8 amino acids. For MHC Class II this is 15 amino acids.
 
-> There are numerous others - check the config file for more.
+## `--max_peptide_length``
 
-Note that you can use the same configuration setup to save sets of reference files for your own use, even if they are not part of the iGenomes resource. See the [Nextflow documentation](https://www.nextflow.io/docs/latest/config.html) for instructions on where to save such a file.
+Specifies the maximum peptide length. By default, for MHC Class I this is 11 amino acids. For MHC Class II this is by default 16
+amino acids.
 
-The syntax for this reference configuration is as follows:
+### `--genome`
 
-```nextflow
-params {
-  genomes {
-    'GRCh37' {
-      fasta   = '<path to the genome fasta file>' // Used if no star index given
-    }
-    // Any number of additional genomes, key is used with --genome
-  }
-}
-```
+This defines against which reference genome the pipeline performs the analysis. The default choice is `GRCh37`, as most clinical labs still rely on `GRCh37` as the human reference genome to use. Available are `GRCh37` and `GRCh38`.
```

Comment:
cf: https://github.com/nf-core/nf-co.re/pull/238
I still still think default should be `GRCh38`, but you can have custom configs for `GRCh37` for your institution ;-)
But we can discuss more.

Reply:
True, could also be changed in 1.1.0 ;-)